tests/mocks/database/population/data/index.ts

```diff
@@ -52,6 +52,7 @@ export interface FacultyData {
   email: string;
   HUID: string;
   jointWith: string | null;
+  notes: string | null;
```

Comment:
@rmainseas I suppose so, but I wasn't sure if there was a specific reason why `jointWith` is set up as `string | null` so I just mirrored that for `notes.` Do you think it makes a difference? I was just copying line 54's optional `jointWith.`

Reply:
So, I'd misremembered how the type system worked. So, `notes?` would be equivalent to `notes: string | undefined` and would throw if not assigned a value, or assigned null. Whereas `notes: string | null` would allow null values. 

So actually, thinking about this, `string | null` is probably correct.